src/pages/home/HeaderView.js

```diff
@@ -54,17 +54,23 @@ const propTypes = {
     }).isRequired,
 
     /** Personal details of all the users */
-    personalDetails: PropTypes.objectOf(participantPropTypes).isRequired,
+    personalDetails: PropTypes.objectOf(participantPropTypes),
```

Comment:
Removing required because these throw unnecessary errors as onyx takes a while to load in these variables. I think all async variables should just be defined as not required.